rails/app/policies/development_policy.rb

```diff
@@ -12,7 +12,11 @@ def create?
   end
 
   def update?
-    !user&.disabled? && (user&.admin? || (user&.verified? && user.developments.include?(record)) || (user&.municipal? && user.developments.include?(record)))
+    if !user&.disabled? && (user&.admin? || (user&.verified? && user.developments.include?(record)) || (user&.municipal? && user.developments.include?(record)))
+      true
+    else
+      [:flag]
+    end
```

Comment:
I don't know if this additional logic does anything. The only people that are allowed to update developments are listed in the conditional which means that the only time the `else` block is reached, it would have to be an unauthorized user. Wouldn't this allow unauthorized users to edit developments if the development has been flagged?

Reply:
No it merely opens the [:flag] attribute to toggling by un-privileged users. See https://stackoverflow.com/a/42016424